kotlinlib/src/mill/kotlinlib/android/AndroidAppKotlinModule.scala

```diff
@@ -18,4 +21,66 @@ import mill.javalib.android.AndroidAppModule
  * [[https://developer.android.com/studio Android Studio Documentation]]
  */
 @mill.api.experimental
-trait AndroidAppKotlinModule extends AndroidAppModule with KotlinModule {}
+trait AndroidAppKotlinModule extends AndroidAppModule with KotlinModule {
```

Comment:
Most of this stuff should probably go into `AndroidAppModule` since non-kotlin android apps will need it too

Reply:
actually not because we have to implement this for jetpack compose 

my thoughts
if we do this in same AndroidAppModule may be it will get some conflict with java and kt both...

Reply:
The point of this task is not to make it support jetpack compose specifically, but to use jetpack compose as an example with requirements that all android apps will need. Nothing here seems specific to Kotlin apps so we should not limit it

Reply:
done